site/sfguides/src/data_teams_with_dbt_core/data_teams_with_dbt_core.md

```diff
@@ -0,0 +1,1447 @@
+<<<<<<< HEAD:site/sfguides/src/data_teams_with_dbt_core/data_teams_with_dbt_core.md
+id: data_engineering_with_dbt_core
+summary: Build your data pipeline with dbt Core & Snowflake
+=======
+id: data_engineering_with_dbt
+summary: Build your data pipeline with dbt & Snowflake
+>>>>>>> dc95377a69000661e445a9e05d1a5d9a4c3bbd1c:site/sfguides/src/data_engineering_with_dbt/data_engineering_with_dbt.md
+categories: Getting Started
+environments: web
+status: Published 
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Getting Started, Data Engineering, dbt, Data Sharing
+authors: Amy Chen, Dmytro Yaroshenko
+
+<<<<<<< HEAD:site/sfguides/src/data_teams_with_dbt_core/data_teams_with_dbt_core.md
+# Accelerating Data Teams with dbt Core & Snowflake
+=======
+# Accelerating Data Teams with dbt & Snowflake
+>>>>>>> dc95377a69000661e445a9e05d1a5d9a4c3bbd1c:site/sfguides/src/data_engineering_with_dbt/data_engineering_with_dbt.md
```

Comment:
merge conflicts breaking the build. Also, the image references seem to be breaking when trying to build.